src/utils/types.ts

```diff
@@ -184,6 +184,19 @@ type EnvVarValue = {
   context: string
 }
 
+export type MinimalAccount = {
```

Comment:
Why not import the type from `@netlify/config`?

Reply:
We don't export it from that module and this is not a public API, so we don't want to make this part of any module's public API.